service/internal/server/server.go

```diff
@@ -458,6 +462,10 @@ func (s OpenTDFServer) Stop() {
 		s.logger.Error("failed to shutdown http server", slog.String("error", err.Error()))
 		return
 	}
+	// Close the listener
+	if s.Listener != nil {
+		s.Listener.Close()
```

Comment:
Should we clear?

Reply:
what do you mean by clear? the listener interface just has accept, close, and addr https://pkg.go.dev/net#Listener

Reply:
Should we set `s.Listener = nil`. I'm guessing the servier object is useless after Stop is called but idk

Reply:
ya i dont think weve tested stopping and then re-starting, but for the listener, its instantiated on start so re-calling start should just create another one and set s.Listener to that